src/utils/browser.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-env browser */
+export function isMobileSafari() {
```

Comment:
can you please write a comment linking the source where you pulled this function from?